WooCommerce/Classes/POS/Utils/Color+WooCommercePOS.swift

```diff
@@ -77,4 +85,8 @@ extension Color {
     static var posTotalsGradientGreen: Color {
         Color(red: 239.0 / 255.0, green: 246.0 / 255.0, blue: 235.0 / 255.0)
     }
+
+    static var posCheckoutBackground: Color {
```

Comment:
Just a question. Do we want all of the colors used on POS be defined here? Even if it's only for a specific view: `posCheckoutBackground`?

Reply:
There is no specific rule, we can use them directly or add them here. I added this one just for the easier readability.